mods/ts/rules/gdi-infantry.yaml

```diff
@@ -8,7 +8,7 @@ E2:
 		Cost: 200
 	Tooltip:
 		Name: Disc Thrower
-		Description: Infantry armed with grenades. \n  Strong vs Buildings, Infantry\n  Weak vs Vehicles
+		Description: Infantry armed with grenades.\n  Strong vs Buildings, Infantry\n  Weak vs Vehicles
```

Comment:
"with special explosive discs"
Also strong vs vehicles, according to http://cnc.wikia.com/wiki/Disk_Thrower